2_delivery_care.do

```diff
@@ -95,30 +101,30 @@ gen country = regexs(1) if regexm(country_year, "([a-zA-Z]+)")
   	if !inlist(name,"Angola2015","Burundi2016","Cameroon2018","Colombia2015","Nepal2016","Tanzania2015"){
 	gen stay = 0 if m15 != .
 	replace stay = 1 if stay == 0 & (inrange(m61,124,198)|inrange(m61,201,298)|inrange(m61,301,399))
-	replace stay = . if inlist(m61,.,299,998)  & !inlist(m15,11,12,96) // filter question, based on m15

```

Comment:
Hi @yining-sun  could you explain why did you delete the condition on missing?

Reply:
Hi @yining-sun , thanks for the re-push. I am following up on this question, could you elaborate on your changes made here?